.github/workflows/python-package.yml

```
@@ -30,7 +30,7 @@ jobs:
         python -m pip install pylint pytest pandas
     - name: Lint with pylint
       run: |
-        pylint --suggestion-mode y
+        $(git ls-files '*.py')
```

```suggestion
        pylint life_expectancy/
```

`$(git ls-files '*.py')` just lists the python files is won't actually run the linter. Running `pylint` + the name of your package should do the trick.

---

I know, it was lack of attention. :)